dashboard/test/models/user_test.rb

```diff
@@ -1054,6 +1099,20 @@ def create_user_with_username(username)
 
     assert_equal [teacher], student.teachers
     assert_equal [], student.students
+
+    # section associations
+    assert_equal [section], student.sections_as_student
+    assert_equal [], teacher.sections_as_student
+    assert_equal [], other_user.sections_as_student
+
+    assert_equal [], student.sections
+    assert_equal [section], teacher.sections
+    assert_equal [], other_user.sections
+
+    # can_pair? method
+    assert_equal true, student.can_pair?
+    assert_equal false, teacher.can_pair?
```

Comment:
This is a result of the teacher not belonging to a section, not the result of the teacher being a teacher, right?